src/fastertransformer/models/gptneox/GptNeoXDecoderLayerWeight.h

```diff
@@ -28,7 +28,7 @@ namespace fastertransformer {
 template<typename T>
 struct GptNeoXDecoderLayerWeight {
 public:
-    GptNeoXDecoderLayerWeight() = delete;
```

Comment:
not sure if any assumption (around the initialization status) breaks here however it increases the complexity of the ways construct GptNeoXDecoderLayerWeight.